tls/s2n_cipher_preferences.c

```diff
@@ -57,6 +57,31 @@ const struct s2n_cipher_preferences cipher_preferences_20190801 = {
     .suites = cipher_suites_20190801,
 };
 
+/* Same as 20190801, but with ECDSA for TLS 1.2 added */
+struct s2n_cipher_suite *cipher_suites_20210831[] = {
```

Comment:
We have documentation for cipher preferences in security policies here: https://github.com/aws/s2n-tls/blob/main/docs/USAGE-GUIDE.md#s2n_config_set_cipher_preferences. Can you update the default_tls13 security policy row to support ECDSA?